Dockerfile

```diff
@@ -60,5 +60,6 @@ VOLUME /superset
 USER superset
 ENTRYPOINT ["/docker-entrypoint.sh"]
 CMD ["gunicorn", "superset:app"]
-HEALTHCHECK CMD ["curl", "-f", "http://localhost:8088/health"]
+# Equivalent of `curl -f http://localhost:8088/health' without pulling additional dependencies.
```

Comment:
What kind of dependencies?

Reply:
Here "additional dependencies" mean "curl". `apt-get install curl` made the image 10MB bigger (537→547MB).

Reply:
so the issue was about failing build process? I do not see removal of line with `apt-get install curl`

Reply:
The build wasn't failing, since healthcheck is not part of the build. Also, you can't see removal of `apt-get install curl` because it has never been added in the first place :)
(at first, i added this as a fix, locally, but then went with py version of the healtheck)